packs/hello-universe-1.1.3/README.md

```diff
@@ -0,0 +1,85 @@
+| `manifests.hello-universe.postgres.tolerations.key`            | The tolerations key to use for the Hello Universe Postgres pods.                         |      `app`                                   | No          |
+| `manifests.hello-universe.ui.tolerations.value`            | The tolerations value to use for the Hello Universe UI pods.                         |      `ui`                                   | No          |
+| `manifests.hello-universe.api.tolerations.value`            | The tolerations value to use for the Hello Universe API pods.                         |      `api`                                   | No          |
+| `manifests.hello-universe.postgres.tolerations.value`            | The tolerations value to use for the Hello Universe Postgres pods.                         |      `postgres`                                | No          |
+
+## Usage
+
+The Hello Universe pack has two presets that you can select:
+- **Disable Hello Universe API** configures Hello Universe as a standalone frontend application. This is the default configuration of the pack. 
+- **Enable Hello Universe API** configures Hello Universe as a three-tier application with a frontend, API server, and Postgres database.
+
+To utilize the Hello Universe pack, create either a [full Palette cluster profile](https://docs.spectrocloud.com/profiles/cluster-profiles/create-cluster-profiles/create-full-profile) or an [add-on Palette cluster profile](https://docs.spectrocloud.com/profiles/cluster-profiles/create-cluster-profiles/create-addon-profile/) and add the pack to your profile. You can select the preset you wish to deploy on the cluster profile creation page.
+
+If your infrastructure provider does not offer a native load balancer solution, such as VMware and MAAS, the [MetalLB](https://docs.spectrocloud.com/integrations/metallb) pack must be included to the cluster profile to help the LoadBalancer service specified in the manifest obtain an IP address.
+
+After defining the cluster profile, use it to deploy a new cluster or attach it as an add-on profile to an existing cluster.
+
+Once the cluster status displays **Running** and **Healthy**, access the Hello Universe application through the exposed service URL along with the displayed port number.
+
+The Hello Universe pack provides parameters for providing pod tolerations. These make it possible to provide [Kubernetes Taints](https://docs.spectrocloud.com/clusters/cluster-management/taints/#taints). You can only apply toleration for the UI, API, and Postgres pods.
+
+The parameters are applied using the `Equal` operator as demonstrated below:
+
+```yaml
+    tolerations:
+    - effect: {{ .Values.ui.tolerations.effect }}
+    key: {{ .Values.ui.tolerations.key }}
+    operator: Equal
+    value: {{ .Values.ui.tolerations.value }}
+```
+
```

Comment:
Is there any chance we could make this section its own H3?
That way it gets a heading and the reader can scan for easily :)

Reply:
Added!